lib/config.js

```diff
@@ -9,6 +9,7 @@ config['messages-client'] = config['messages-client'] || {};
 config['messages-client'].clients = config['messages-client'].clients || [];
 config['messages-client'].maxMessageRetrievalRetries =
   config['messages-client'].maxMessageRetrievalRetries || 3;
+config['messages-client'].stopOnMaxRetries = true;
```

Comment:
I think this would be unclear to anyone not involved in writing this code. My first thought would be "it says max retries, why wouldn't you stop?". What isn't clear here is that what we're really doing is configuring when a *notification* of a failure will go out. So maybe we're talking about "maxMessageRetrievalFailuresBeforeNotify" not "maxRetries"... and "stopMessageRetrievalAfterFailureNotification". Of course, if you can shorten those up without losing the meaning, that would be great. :)